index.js

```diff
@@ -113,6 +113,7 @@ module.exports = function(version) {
             var type = step.maneuver.type;
             var modifier = step.maneuver.modifier;
             var mode = step.mode;
+            var side = step.driving_side;
```

Comment:
If the input comes from OSRM v5.0–v5.13, `driving_side` will be undefined. [`String.indexOf()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf) apparently returns −1 when given no argument, so `omitSide` ends up false, preserving the previous behavior. As long as we can rely on this behavior of `indexOf()`, it should be no problem, but can would you mind including a comment to this affect?